doc/paper/espnet-se++/paper.md

```diff
@@ -562,8 +195,6 @@ The `ESPnetEnhS2TModel` takes a front-end `enh_model`, and a back-end `s2t_model
 	    ...
 	):
 	
-
-<!-- The front-end model currently should be an SE model, and the back-end model could be ASR, SLU, ST, and SD models. -->
 The `forward` function of the class follows the general design in ESPnet2:
 	 
 	 def  forward(self, speech_mix, speech_ref, ...)
```

Comment:
Since the description about `Trainer` is removed, it might be better to add a hyperlink to the definition of `Trainer` below:

```which processes speech and only returns losses for `Trainer` to update the model. ```

Reply:
updated with this hyperlink:
https://github.com/espnet/espnet/blob/master/espnet2/train/trainer.py#L87-L108